src/egon/data/datasets/renewable_feedin.py

```diff
@@ -34,12 +34,11 @@ def weather_cells_in_germany(geom_column='geom'):
     cfg = egon.data.config.datasets()['renewable_feedin']['sources']
 
     return db.select_geodataframe(
-        f"""SELECT a.w_id, a.geom_point, a.geom
+        f"""SELECT w_id, geom_point, geom
         FROM {cfg['weather_cells']['schema']}.
-        {cfg['weather_cells']['table']} a,
-        {cfg['vg250_sta_union']['schema']}.
-        {cfg['vg250_sta_union']['table']} b
-        WHERE ST_Intersects(ST_Transform(b.geometry, 4326), a.geom)""",
+        {cfg['weather_cells']['table']}
+        WHERE ST_Intersects('SRID=4326;
+        POLYGON((5 56, 15.5 56, 15.5 47, 5 47, 5 56))', geom)""",
```

Comment:
Could you increase the version number of `RenewableFeedin` to make sure that all tasks which depend on this changes are executed?

Reply:
Done. Now the version number is 0.0.1